keyboards/ploopyco/knob/config.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2025 Colin Lam (Ploopy Corporation)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define POINTING_DEVICE_HIRES_SCROLL_ENABLE 0
```

Comment:
Not sure what the goal here is.

If this needs hi res scroll to be disabled, #undef'ing it may be a better option